config/environments/development.rb.SAMPLE

```diff
@@ -39,7 +39,7 @@ Foodsoft::Application.configure do
   # Mailcatcher config, start mailcatcher from console with 'mailcatcher'
   # Mailcatcher can be installed by gem install mailcatcher
   config.action_mailer.delivery_method = :smtp
-  config.action_mailer.smtp_settings = { address: "localhost", port: 1025 }
+  config.action_mailer.smtp_settings = { address: "mailcatcher", port: 1025 }
```

Comment:
I like moving system-dependent config to environment variables. But since the defaults for each rails env are different, I would think this still needs to go into the environment's settings, right?